tests/Doctrine/Tests/Models/Issue5989/Issue5989Employee.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Doctrine\Tests\Models\Issue5989;
+
+/**
+ * @Entity
+ * @Table(name="issue5989_employees")
+ */
+class Issue5989Employee extends Issue5989Person
+{
+    /**
+     * @column(type="simple_array", nullable=true)
```

Comment:
Yea, I think it was copy paste from some existing models you have there. There are quite a few with lowercase. But I'll fix these.